Traceback (most recent call last):
  File "/home/qcqced/바탕화면/ML_Test/UPPPM/train.py", line 53, in <module>
    main('upppm_config.json', CFG)
  File "/home/qcqced/바탕화면/ML_Test/UPPPM/train.py", line 34, in main
    getattr(train_loop, cfg.loop)(cfg)  # init object
  File "/home/qcqced/바탕화면/ML_Test/UPPPM/trainer/train_loop.py", line 35, in train_loop
    loader_train, loader_valid, train, valid_labels = train_input.make_batch(fold)
  File "/home/qcqced/바탕화면/ML_Test/UPPPM/trainer/trainer.py", line 33, in make_batch
    tmp_valid = valid.explode('scores').tolist()
  File "/home/qcqced/anaconda3/lib/python3.9/site-packages/pandas/core/generic.py", line 5575, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'tolist'